src/storage/src/hummock/sstable_store.rs

```diff
@@ -149,6 +149,9 @@ impl SstableStore {
         let meta_cache_v2 = HybridCacheBuilder::new()
             .memory(meta_cache_capacity)
             .with_shards(1)
+            .with_weighter(|_: &HummockSstableObjectId, value: &Box<Sstable>| {
+                u64::BITS as usize / 8 + value.estimate_size()
```

Comment:
`u64::BITS / 8` == `sizeof::<u64>`?

Any reason not to use `sizeof::<Box<Sstable>>` or `sizeof::<HummockSstableObjectId>`?

Reply:
IMO, the main cause the build procedures for the compute node and the compactor node are not aligned well. I'll refactor them later.